fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaConstants.java

```diff
@@ -29,6 +28,7 @@
 
     // Group of privilege grants used for FHIRUSER access
     public static final String FHIR_USER_GRANT_GROUP = "fhiruser";
+    public static final String FHIR_BATCH_GRANT_GROUP = "fhirbatch";
```

Comment:
is it ever used anywhere?

Reply:
no but it should be! great find

Reply:
Fixed, it's now used

Reply:
It makes me wonder if we should have a FHIR_OAUTH_GRANT_GROUP as well, but I think this PR already has enough in it.
We'll need to remember to update the diagrams with this new role (e.g. in the fhir-persistence-schema/docs and update the guides to indicate that the batch user must be in the fhirbatch role for this feature to work).
Personally, I might have saved that for a separate PR, but I'm OK with including it here.